misc/python/materialize/parallel_workload/action.py

```diff
@@ -800,7 +800,7 @@ def run(self, exe: Executor) -> bool:
 
 class AlterKafkaSinkFromAction(Action):
     def run(self, exe: Executor) -> bool:
-        if exe.db.scenario in (Scenario.Kill, Scenario.ZeroDowntimeDeploy):
+        if exe.db.scenario in (Scenario.ZeroDowntimeDeploy,):
```

Comment:
Why was this changed?

Reply:
Not an interesting reason: I was trying to re-enable this for as many scenarios as possible, and it seemed plausible to me that this or recent changes fixed whatever caused us to exclude it earlier.

But you definitely have more context than I do, and if you prefer I'm happy to leave this line alone!

Reply:
I think it will be slightly flaky with this change.

Reply:
Aight, I'll change it back!

Is it easy to explain why `alter sink` is not expected to work in the kill scenario? I'm curious!

Reply:
https://github.com/MaterializeInc/database-issues/issues/8421
> I believe what happens is that we have interrupted queries during a kill and thus are not aware of the exact state of a table, thus we can't replace it.